### PR TITLE
chore: update frontend environment variables

### LINKED ISSUE #2994: Frontend environment variables
## Context

When the tech writers went through the frontend environment variables, we identified some gaps that we want to close. 

## Todos
- Environment variables
  - [ ] remove default for GRAPHQL_URL and make it required (i.e. app doesn't start if it's not set)
  - [ ] remove default for OPEN_PAYMENTS_URL and make it required
  - [ ] set NODE_ENV default to `production`
  - [ ] set ENABLE_INSECURE_MESSAGE_COOKIE to `true` and remove check for NODE_ENV
- Code
  - [ ] add KRATOS_ADMIN_URL to `variables` in packages/frontend/app/lib/envConfig.server.ts
  - [ ] change `start` command to `remix-serve ./build/index.js` in packages/frontend/package.json
- Docs
  - [ ] Update docs accordingly once _Environment variables_ and _Code_ changes have been done

### PR DESCRIPTION
**Pull request by oana-lolea**
## Changes proposed in this pull request

- Updated frontend environment variables: `GRAPHQL_URL`, `OPEN_PAYMENTS_URL`, `ENABLE_INSECURE_MESSAGE_COOKIE`, `NODE_ENV`, `KRATOS_ADMIN_URL`.
- Changed start command in `package.json`.
- Updated docs with the new changes.

## Context

For this issue, I made the following changes:
- Made `GRAPHQL_URL` and `OPEN_PAYMENTS_URL` mandatory and removed their default values.
- Set `NODE_ENV ` to `production` in docker files.
- Set default value for `ENABLE_INSECURE_MESSAGE_COOKIE` to `true` in docker files and removed it from `start` script in `package.json`
- Added `KRATOS_ADMIN_URL` to `variables` in `envConfig.server.ts`.
- Updated `frontend-variables.mdx`.

Fixes #2994

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Make sure that all checks pass
- [ ] Bruno collection updated (if necessary)
- [ ] Documentation issue created with `user-docs` label (if necessary)
- [ ] OpenAPI specs updated (if necessary)

### CODE REVIEW

File: packages/documentation/src/partials/frontend-variables.mdx [hunk 1]
```diff
@@ -2,11 +2,11 @@ import { LinkOut } from '@interledger/docs-design-system'
...
-| `OPEN_PAYMENTS_URL`              | frontend.serviceUrls.OPEN_PAYMENTS_URL | `http://localhost:3003/`        | Open Payments API Endpoint                                                                                                                                                              |
-| `PORT`                           | frontend.port                          | `3005`                          | Port from which to host the Remix app                                                                                                                                                   |
-| `ENABLE_INSECURE_MESSAGE_COOKIE` |                                        | _undefined_                     | Values of `true`, `t`, `1` will not use a `secure` message cookie which is required for flash messages to work over http. When not set, the secure flag is set according to `NODE_ENV`. |
+| Variable                         | Helm Value Name                        | Default     | Description                                                                                                                                                                             |
```
Comment (mkurapov) [inline packages/documentation/src/partials/frontend-variables.mdx:5]: Let's tag @melissahenderson for this PR so she can make docs changes as necessary

File: packages/frontend/app/lib/message.server.ts [hunk 2]
```diff
@@ -21,9 +21,10 @@ export const messageStorage = createCookieSessionStorage<
     sameSite: 'lax',
     maxAge: ONE_MINUTE_IN_S,
     secrets: ['MY_SUPER_SECRET_TOKEN'],
-    secure: process.env.ENABLE_INSECURE_MESSAGE_COOKIE
-      ? !parseBool(process.env.ENABLE_INSECURE_MESSAGE_COOKIE)
-      : process.env.NODE_ENV === 'production'
+    secure:
+      process.env.ENABLE_INSECURE_MESSAGE_COOKIE !== undefined
+        ? !parseBool(process.env.ENABLE_INSECURE_MESSAGE_COOKIE)
+        : false
```
[the remark this answers is not in the document]
Comment (oana-lolea) [reply packages/frontend/app/lib/message.server.ts:27]: The variable is supposed to enable _insecure_ cookies, so shouldn't the default value be false for the secure property?

[the remark this answers is not in the document]
Comment (mkurapov) [reply packages/frontend/app/lib/message.server.ts:27]: The way this is written, as an argument to `createCookieSessionStorage({ cookie: { secure }})`, we either use `!ENABLE_INSECURE_MESSAGE_COOKIE` or we fallback to `true`, to specify it is `secure` by default

**Comment by melissahenderson** [comment]
Related to #2994

**Comment by melissahenderson** [comment]
Doc updates are ready to go in PR #3042 3042, but I'm waiting for this issue and #2994 to be merged first. (Please let me merge my PR when it's time, there are prettier issues I need to fix first).

**Review by mkurapov — APPROVED** [review]

**Review by njlie — APPROVED** [review]

**Comment by oana-lolea** [comment]
@melissahenderson merged!

**Pull request merged**